### DISCUSSION TITLE
Audio Only?

### DISCUSSION THREAD

**Discussion by aclater** [automatic-ripping-machine/automatic-ripping-machine#793] [General]
Is there any demand for an "audio only" option from an install perspective? ARM has grown significantly around ripping DVD/BRay, but its functionality that I simply don't use. 

Would there be interest in a lighter weight installation option for those only using to rip audio?

**Comment by microtechno9000** [comment]
at the moment the focus is on getting the DVD and bluray ripping into a good state.

It could be possible for a future enhancement to incorporate this feature.  However at the moment is not planned.
